docs/examples/rbac/go/app.go

```diff
@@ -46,7 +46,9 @@ o.RegisterClass(reflect.TypeOf(Repository{}), nil)
 o.RegisterClass(reflect.TypeOf(User{}), NewUser)
 // docs: end-register
 
-o.LoadFiles([]string{"main.polar"})
+if err := o.LoadFiles([]string{"main.polar"}); err != nil {
+	panic(err)
+}
```

Comment:
Ah yeah I added this actually because I was getting a silent error here (which had to do with me not running `make` in the go directory for a while). Happy to revert this piece for simplicity/conciseness.